.github/linters/.textlintrc

```diff
@@ -0,0 +1,41 @@
+{ 
+  "rules": {
+    "terminology": {
+      "severity" : "warning",
+      "exclude": [
+        "back[- ]end(\\w*)",
+        "bug[- ]fix(es)?",
+        "client ?side",
+        "end-?user(s)?",
+        "indexes\\b",
+        "front[- ]end(\\w*)",
+        "Mac ?OS",
+        "OS X",
+        "readme(s)?",
+        "server ?side",
+        "smartphone(s)?",
+        "spell-?check(er|ing)?",
+        "style-?sheet(s)?",
+        "web-?page(s)?",
+        "web[- ]?site(s)?",
+        "white[- ]space",
+        "ZIP"
+      ],
+      "terms": [
+        ["backend(\\w*)", "back end$1"],
+        ["bug[- ]fix", "bugfix"],
+        ["bug-?fixes", "bug fixes"],
+        ["indices\\b", "indexes"],
+        ["frontend(\\w*)", "front end$1"],
```

Comment:
I strongly disagree with changing frontend to front end, We have an entire Frontend Developer Guide. Backend should also be one word, but I don't care about that as much because I avoid using it (Steve Johnson influence there).

Reply:
Removed the rule